data_steward/retraction/retract_data_bq.py

```diff
@@ -277,59 +171,38 @@ def get_retraction_queries(client: BigQueryClient,
         or if RDR data needs to be kept intact. Can take the values 'rdr_and_ehr' or 'only_ehr'
     :return: list of queries
     """
-    LOGGER.info(f'Checking existing tables for {client.project}.{dataset_id}')
-    existing_tables = [
-        table.table_id
-        for table in client.list_tables(f'{client.project}.{dataset_id}')
-    ]
-    tables_to_retract = [
-        table for table in existing_tables
-        if any(col.name == 'person_id' for col in client.get_table(
-            f'{client.project}.{dataset_id}.{table}').schema)
-    ]
+    tables_to_retract = get_tables_to_retract(client,
+                                              dataset_id,
+                                              retraction_type=retraction_type,
+                                              hpo_id=hpo_id)
+
     LOGGER.info(
         f"Tables to retract in {dataset_id}:\n"
         f"Tables with person_id column... {', '.join(tables_to_retract)}\n"
-        f"If it's a non-deid dataset, {FACT_RELATIONSHIP} will be retracted too.\n"
+        f"If it's a non-deid dataset, {FACT_RELATIONSHIP} will be retracted too."
     )
 
-    person_id = RESEARCH_ID if ru.is_deid_dataset(dataset_id) else PERSON_ID
+    person_id = RESEARCH_ID if is_deid_dataset(dataset_id) else PERSON_ID
 
     queries = []
 
     for table in tables_to_retract:
-
-        if skip_retraction(client, dataset_id, table, retraction_type):
-            continue
-
-        only_ehr_condition = get_only_ehr_condition(
-            client, dataset_id,
-            table) if retraction_type == RETRACTION_ONLY_EHR else ''
-
-        if not skip_sandboxing:
-            q_sandbox = JINJA_ENV.from_string(RETRACT_QUERY).render(
-                sandbox=True,
-                sb_table=f'retract_{dataset_id}_{table}',
+        for create_sandbox in [False] if skip_sandboxing else [True, False]:
```

Comment:
This is functionally correct but I was a little confused when reading the loop out of context with the query.  I think it is clearer to say:
```python
for action in ['delete'] if skip_sandboxing else ['sandbox', 'delete']:
```
And then update the loop in the Jinja env to check against the action value.  
Just from reading the PR, this gives me a better idea of the loop intention.  I didn't realize it was choosing a SQL action based on this line.